src/displayapp/screens/settings/SettingAirplaneMode.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <lvgl/lvgl.h>
+
+#include "components/settings/Settings.h"
+#include "displayapp/screens/Screen.h"
+
+namespace Pinetime {
+
+  namespace Applications {
+    namespace Screens {
+
+      class SettingAirplaneMode : public Screen {
+      public:
+        SettingAirplaneMode(DisplayApp* app, Pinetime::Controllers::Settings& settingsController);
+        ~SettingAirplaneMode() override;
+
+        void UpdateSelected(lv_obj_t* object, lv_event_t event);
+
+      private:
+        static constexpr std::array<const char*, 2> options = {" No", " Yes"};
```

Comment:
I think the "Yes" option should be above "No"

Reply:
+1

Reply:
I disagree.

I did a HCI/UX paper prototype study to test the following options before selecting this option (n=16):
* Yes/No 56% of respondents performed correctly
* On/Off 25% of respondents performed correctly
* Enable/Disable 19% of respondents performed correctly

Regarding the order, the existing UX displays the default choice first and "No airplane mode" is the default mode. Therefore the order should be "No" then "Yes" for consistency.

Reply:
I don't really understand what you mean. Do you mean that if a person goes to enable the airplane mode, only 19% would be able to press the correct button to enable it? I highly doubt that is the case. "Airplane mode. Yes/No" doesn't really make sense in a literal sense either.

Having the default option first sounds reasonable, but somehow having "No" above "Yes" doesn't feel right. Unfortunately I'm not able to explain why.

This could also just be a switch widget.